packages/office-ui-fabric-react/src/components/Layer/Layer.tsx

```diff
@@ -1,17 +1,13 @@
-import * as ReactDOM from 'react-dom';
 import { styled } from '../../Utilities';
 import {
   ILayerProps,
   ILayerStyleProps,
   ILayerStyles
 } from './Layer.types';
 import { LayerBase } from './Layer.base';
-import { PortalLayerBase } from './PortalLayer.base';
 import { getStyles } from './Layer.styles';
 
-const portalSupport: boolean = !!ReactDOM.createPortal;
-
 export const Layer = styled<ILayerProps, ILayerStyleProps, ILayerStyles>(
-  portalSupport ? PortalLayerBase : LayerBase,
```

Comment:
Just set this flag to false for now?

Reply:
If we want to create a release without the change, that would be the least invasive change I feel and would make it easier for forward fix the components that might have changed behavior. 

You should be able to validate with a conditional breakpoint that changes this var